setup.py

```diff
@@ -13,7 +13,7 @@
         'python-dateutil',
         'Django>2.2,<2.3',
         'flattentool>=0.5.0',
-        'libcove>=0.18.0'
+        'libcove>=0.19.0'
```

Comment:
What about pinning this to `==0.19.0` - is there any harm in doing os?

Reply:
The downside is that doing the upgrade takes more work. These upgrades don't usually break things, and we've been doing some work to avoid it happening unexpetedly in future, by [running the lib-cove-bods for each new lib-cove commit](https://github.com/OpenDataServices/lib-cove/blob/main/.github/workflows/integration.yml).